### ISSUE TITLE
Create a checklist for manual testing of documents

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/onrr.gov-site#3209]
I might create multiple checklists based on document type. 

The goal of these checklists will be to better inform doc owners of accessibility issues that are not identified in automated testing and require manual testing.

For the ODDD team, we can also use these checklists when manual testing docs. When a doc fails one of the check points, we can then send it back to the doc owner to fix.

**Comment by cthomasONRR** [comment]
@eelzi-ONRR This is ready for your review. Document [here](https://doimspp-my.sharepoint.com/:w:/r/personal/thomasc_mms_gov/Documents/Documents/Manual%20Accessibility%20Testing%20of%20Documents%20Checklist.docx?d=w5d864767e1d542b18599de5ac818ac1e&csf=1&web=1&e=JX92dU)

**Comment by cthomasONRR** [comment]
@lpgoldstein Do you have time to review the wiki? Erin and I went over content, need new eyes for typos.
https://github.com/DOI-ONRR/onrr.gov-site/wiki/Accessibility-Guidance#manual-accessibility-testing-of-documents-checklist

Really happy how the headings/checklist items appear in the ToC
![image](https://github.com/user-attachments/assets/b392e414-7d78-41ed-bb89-bd1bb7c8de74)

**Comment by lpgoldstein** [comment]
@cthomasONRR looks great! Fixed a few possessive things, changed utilize to use and changed Web page to web page. Thanks for doing this!
